### ISSUE TITLE
Finger tutorial still has reference to .tap files

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4679]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#4679|
|Type|defect|
|Created|2010-10-08 18:58:31Z|
|Branch|https://github.com/twisted/twisted/tree/finger-tutorial-4679|

core/howto/tutorial/configuration.xhtml states that ".tap files are covered in Writing a twistd Plugin"; this should be rephrased to state that twistd plugins are covered.

Later on in the document, ServiceMaker references "finger.tap". A comment next to this noting that this is a module reference (finger/tap.py) rather than a filename may also be helpful. Or maybe that file shouldn't be tap.py anymore.

Attachments:

* [finger-tutorial-4679.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/552/552a228ca59647ff1e7514e0ee85ccf7b549bd75/187f6482211f4fd76c721727c13b38aaab9b7c4b.diff) (1175 bytes) - added by lvh on 2010-10-10 11:43:04Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4679 4679
type__defect defect
reporter__itamar itamar
priority__low low
milestone__ 
branch__branches_finger_tutorial_4679 branches/finger-tutorial-4679
branch_author__lvh lvh
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1286564311000000 1286564311000000
changetime__1290275981000000 1290275981000000
version__None None
owner__lvh lvh
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_1"></a>|@lvh removed owner|
|-|-|

lp:~lvh/twisted/finger-tutorial-4679

diff attached

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_2"></a>|@jesstess set owner to @lvh|
|-|-|

Thanks for working on this, lvh. Can you put 'ServiceMaker', 'finger.tap', and 'finger/tap.py' in a fixed-width font in the sentence "Note that the second argument to ServiceMaker, 'finger.tap', is a reference to a module (finger/tap.py), not to a filename."

Other than that and a NEWS file, please deploy!

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_3"></a>|@lvh commented|
|-|-|

(In [30284]) Branching to 'finger-tutorial-4679'

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_4"></a>|@lvh set status to `closed`|
|-|-|

(In [30289]) Merge finger-tutorial-4679: removed reference to .tap files in finger tutorial

Author: lvh
Reviewers: jesstess
Fixes: [#4679](https://github.com/twisted/twisted/issues/4679)
